app.rb

```diff
@@ -0,0 +1,117 @@
+require "sinatra/base"
+require "sinatra/reloader"
+require "sinatra/activerecord"
+require "rack-flash"
+
+require_relative "./models/user"
+require_relative "./models/peep"
+
+class Application < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  configure do
+    enable :sessions
+    use Rack::Flash
+  end
+
```

Comment:
Appreciate how a lot of functionality is being achieved through what seems like a reasonably small amount of, but very clear routes.